src/ORM/Query.php

```diff
@@ -31,6 +33,41 @@
  * into a specific iterator that will be responsible for hydrating results if
  * required.
  *
+ * @see CollectionInterface For a full description of the collection methods supported by this class
+ * @method CollectionInterface each(callable $c) Passes each of the query results to the callable
```

Comment:
Indeed a little long... my concern was less a technical one, it's just that it would be different to all other places in the DocBlocks where the FQN will always be used.